docker-build-env/Dockerfile

```diff
@@ -11,6 +11,8 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y --no-install-rec
         googletest \
         libboost-all-dev \
         libssl-dev \
+        # For llvm-symbolizer (used by ASAN).
+        llvm \
```

Comment:
Wasn't ASAN already enabled in previous builds?
Is there a reason why it is needed now?

Reply:
Do you want to add this to your comment?
Iirc Ivan already asked the same. So it is not obvious.

Reply:
Done.